### PR TITLE
feat: take locale relavant while formatting date in /bookings

### LINKED ISSUE #7680: Missing translation in the booking overview
Missing translation and wrong time format in the booking overview (German)
Bildschirm­foto 2023-03-12 um 21 17 47

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

Fixes #7680

after:
Screenshot 2023-03-13 at 18-24-02 Buchungen Cal com

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by Udit-takkar — CHANGES_REQUESTED** [review]
Screenshot 2023-03-13 at 8 37 26 PM

Recurring events dates are pending. Everything else looks fine

**Comment by G3root** [comment]
@Udit-takkar pushed those changes 

Screenshot 2023-03-13 20:52:12

**Review by Udit-takkar — APPROVED** [review]
LGTM 🚀🚀🚀🚀

File: apps/web/components/booking/BookingListItem.tsx [hunk 1]
```diff
@@ -186,7 +189,9 @@ function BookingListItem(booking: BookingItemProps) {
     );
   };
 
-  const startTime = dayjs(booking.startTime).format(isUpcoming ? "ddd, D MMM" : "D MMMM YYYY");
+  const startTime = dayjs(booking.startTime)
+    .locale(language)
+    .format(isUpcoming ? "ddd, D MMM" : "D MMMM YYYY");
```
Comment (roae) [inline apps/web/components/booking/BookingListItem.tsx:194]: have we ever tried use the localizedFormat dayjs plugin to display this?

imagen

**Review by roae — APPROVED** [review]
👍

**Pull request merged**